pkg/apic/specparser.go

```diff
@@ -13,29 +13,34 @@ import (
 	"gopkg.in/yaml.v2"
 )
 
-type specProcessor interface {
-	getEndpoints() ([]EndpointDefinition, error)
+// SpecProcessor -
+type SpecProcessor interface {
```

Comment:
Maybe not now, but considering the spec parser is not just used by apic, maybe we move it to its own package, util/specparser/*.go?

Reply:
Yes

Reply:
logged ticket https://jira.axway.com/browse/APIGOV-22680 to take care of this plus context creation/use